receiver/flinkmetricsreceiver/README.md

```diff
@@ -0,0 +1,45 @@
+# FlinkMetrics Receiver
+
+This receiver uses Flink's [REST API](https://nightlies.apache.org/flink/flink-docs-release-1.14/docs/ops/metrics/#rest-api-integration) to collect Jobmanager, Taskmanager, Job, Task and Operator metrics.
```

Comment:
This should be moved below the status table

Reply:
updated